README.md

```diff
@@ -0,0 +1,36 @@
+# Terraform Static Website Infrastructure
+
+This repository presents the use of a module with an infrastructure for hosting a React application with AWS S3 and AWS CloudFront with the possibility of creating domain aliases with AWS Route53 and generating a certificate with Amazon Certificate Manager.
+
+## Requirements
+
+| Name | Version |
+|---|---|
+| terraform  | 1.2.3 |
+| aws-cli    | 2.4.2 |
+
+## Providers
+
+| Name | Version |
+|---|---|
+| aws  | >= 4.19.0 |
+
+## Usage
+
+Assuming you have already configured a profile in aws-cli with proper permissions to run terraform. You should change values `dev.tfvars` file in environments folder.

```

Comment:
The first sentence could serve as a disclaimer for this section. I added a few connectors to the latter sentence.

```suggestion
**DISCLAIMER:** The following commands assume you have already configured a
profile in aws-cli with proper permissions to run this Terraform configuration.

You should change the values in the `dev.tfvars` file, in the `environments`
folder.
```